fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -34,7 +66,8 @@ def __init__(self,
                  quality_metric):
         self.learner = learner
         if (not isinstance(disparity_metric, DemographicParity) and
-                not isinstance(disparity_metric, BoundedGroupLoss)):
+                not isinstance(disparity_metric, BoundedGroupLoss) and
+                not isinstance(disparity_metric, Moment)):
```

Comment:
This should become the only check we need?

Reply:
Yes.